src/assets/SubstanceList.jsx

```diff
@@ -0,0 +1,351 @@
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 10,
+    name: "DMT",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 11,
+    name: "2C-B",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 12,
+    name: "Benzodiazepines",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 13,
+    name: "Cafeine",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 14,
+    name: "GHB",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 15,
+    name: "Opioids",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+  {
+    id: 16,
+    name: "Alcohol",
+    active: false,
+    combinations: {
+      1: "Synergy",
+      2: "Caution",
+      3: "Caution",
+      4: "Synergy",
+      5: "Caution",
+      6: "Synergy",
+      7: "Caution",
+      8: "Synergy",
+      9: "Synergy",
+      10: "Synergy",
+      11: "Synergy",
+      12: "Dicrease",
+      13: "No Synergy",
+      14: "Synergy",
+      15: "Synergy",
+    },
+  },
+];
```

Comment:
This could be in a .json file. You can define it inside a data folder on src. Then you can directly import it or simulate a fetch inside the component. The structure could be something like this:

```
[
  {
    id: 1,
    name: 'Cannabis',
    combinations: [
      {
        id: 2,
        result: 'Caution',
        description: 'Cannabis has an unexpectedly strong and somewhat unpredictable synergy with psychedelics.'
      },
      ...
    ]
  },
  ...
]
```

Reply:
What would be the use of "label" in this case?
Isn't is better if combinations is an object instead of an array here?

Reply:
The purpose of `label` is to display a prettier name in the UI. Checking again, maybe we can have 'cannabis' as the substance `id` and then have `name` or `label` to display it the actual name. I think it will be more clear.